File: Python/tests/test_ra_noRA.py
import pytest

from hbayesdm.models import ra_noRA


def test_ra_noRA():
    _ = ra_noRA(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_ra_noRA.R
context("Test ra_noRA")
library(hBayesDM)

test_that("Test ra_noRA", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(ra_noRA(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
